<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSettings</class>
 <widget class="QWidget" name="GeneralSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Map name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="mapNameEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Map description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="mapDescriptionEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Author</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="authorEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Author contact (e.g. email)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="authorContactEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Map Creation Time</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="mapCreationDateTimeEdit">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Map Version</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="mapVersionEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QSpinBox" name="heroLevelLimit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="heroLevelLimitCheck">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Limit maximum heroes level</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Difficulty</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="diffRadio1">
        <property name="text">
         <string notr="true">1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="diffRadio2">
        <property name="text">
         <string notr="true">2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="diffRadio3">
        <property name="text">
         <string notr="true">3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="diffRadio4">
        <property name="text">
         <string notr="true">4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="diffRadio5">
        <property name="text">
         <string notr="true">5</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
